inspirehep/modules/records/serializers/schemas/json.py

```diff
@@ -44,8 +45,17 @@ class RecordSchemaJSONUIV1(Schema):
 
 class MetadataAuthorsSchemaV1(Schema):
 
-    authors = fields.Raw(dump_only=True)
+    authors = fields.Method('get_authors')
     collaborations = fields.Raw(dump_only=True)
+    supervisors = fields.Method('get_supervisors')
+
+    def get_authors(self, data):
+        record = LiteratureRecord(data)
+        return record.authors or missing
```

Comment:
it's better to remove the missing cause we are always expecting authors key on the UI

Reply:
`authors` is not required, so there isn't always an author. Sometimes there's a `corporate_author` instead.

Reply:
@drjova then I have to always return empty list from the prperties, right?